### PR TITLE
Fix prop-types and create-react-class warning

### PR DESCRIPTION
**Pull request by ddcat1115**
same as https://github.com/react-component/dialog/pull/56

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,9 +53,9 @@
     "jquery": "^1.11.2",
     "pre-commit": "1.x",
     "rc-tools": "5.x",
-    "react": "15.3.x",
-    "react-addons-test-utils": "15.3.x",
-    "react-dom": "15.3.x",
+    "react": "15.5.0",
```
Comment (afc163) [inline package.json]: 改成 `^15.5.0`。

File: package.json [hunk 2]
```diff
@@ -53,9 +53,9 @@
...
+    "react-addons-test-utils": "15.5.0",
```
Comment (afc163) [inline package.json]: react-addons-test-utils 可以去掉

`react-addons-test-utils` => `react-dom/test-utils`

File: package.json (hunk 1 above)
Comment (ddcat1115) [inline package.json]: finally...

**Comment by afc163** [comment]
need a patch release @yiminghe

**Pull request merged**